index.html

```diff
@@ -3829,18 +3511,369 @@ <h2>
+        particular member and what to do when a type does not match what is
+        expected.
       </p>
-    </section>
+      <ul class="index" data-sort="">
+        <li>[[!URL]] defines the following terms:
+          <ul>
+            <li>
+              <dfn><a data-cite="!URL#concept-url">URL</a></dfn>
+            </li>
+            <li>
+              <dfn data-lt="parse|parsing|URL parsing"><a data-cite=
+              "!URL#concept-url-parser">URL parser</a></dfn>
+            </li>
+          </ul>
+        </li>
+        <li>[[!SCREEN-ORIENTATION]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite=
+              "!SCREEN-ORIENTATION#dfn-default-orientation"><dfn>default
+              orientation</dfn></a>
+            </li>
+            <li>
+              <a data-cite=
+              "!SCREEN-ORIENTATION#idl-def-OrientationLockType"><dfn>OrientationLockType</dfn></a>
+            </li>
+          </ul>
+        </li>
+        <li>[[!CSS-SYNTAX-3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="CSS-SYNTAX-3#parse-component-value"><dfn>parse a
+              component value</dfn></a>
+            </li>
+          </ul>
+        </li>
+        <li>[[!CSP3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite=
+              "!CSP3#directive-manifest-src"><dfn>manifest-src</dfn>
+              directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-img-src"><dfn>img-src</dfn>
+              directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-default-src"><dfn>default-src</dfn>
+              directive</a>
+            </li>
+          </ul>
+        </li>
+        <li>[[!SECURE-CONTEXTS]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="!secure-contexts#is-origin-trustworthy"><dfn>Is
+              origin potentially trustworthy</dfn></a>
+            </li>
+          </ul>
+        </li>
+        <li>[[!SERVICE-WORKERS-1]] defines the following terms:
+          <ul>
+            <li>
+              <a href=
+              "https://w3c.github.io/ServiceWorker/#start-register-algorithm"><dfn>
+              Start Register</dfn></a>
+            </li>
+            <li>
+              <a href=
+              "https://w3c.github.io/ServiceWorker/#dfn-scope-url"><dfn>scope
+              URL</dfn></a>
+            </li>
+            <li>
+              <a href=
+              "https://w3c.github.io/ServiceWorker/#dfn-update-via-cache"><dfn>ServiceWorkerUpdateViaCache</dfn></a>
+            </li>
+          </ul>
+        </li>
+        <li>[[!ECMAS-402]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite=
+              "!ecma-402#sec-6.2.2"><dfn>IsStructurallyValidLanguageTag</dfn></a>
```

Comment:
Uppercase ECMA-402 otherwise:

>[ECMA-402] is referenced in 2 ways: ('ecma-402', 'ECMAS-402'). This causes duplicate entries in the References section.